setup.py

```diff
@@ -167,7 +167,7 @@ def build_extensions(self):
 
 # Setup RPATH on Linux and MacOS
 if sys.platform == "darwin":
-    extra_link_args = ["-rpath", ".", "-rpath", "@loader_path/head/lib"]
+    extra_link_args = ["-Wl,-rpath,@loader_path/head/lib"]
```

Comment:
Again: Is this only true for gcc or does it work for clang, too?

Reply:
Ah, I get it now. I saw somewhere that one should still include explicitly "." into the rpath, so better use 
`["-Wl,-rpath,.", "-Wl,-rpath,@loader_path/head/lib"]`

Reply:
Resolved.